### ISSUE TITLE
Can't use the vanilla attributes.

### ISSUE THREAD

**Issue by EnergyREX** [HyperPigeon/Eldritch-Mobs#47]
Hello, I was testing some things of the mod and I found that executing the commands as like, summon-eldritch, summon-elite or summon-ultra you can't use the attributes like {NoAI:1}. When you execute the command, 
for example "/summon_ultra minecraft:cat ~ ~ ~ {NoAI:1}" It summons the cat, but an error is shown: commands.summon_ultra.failed. The cat, in this case when It's summoned, is a normal cat, with no new drops or HP multiplier. If is a vanilla minecraft boss, the bossbar is not shown.

**Comment by HyperPigeon** [comment]
Have you added cats to the list of allowed eldritch mobs?

**Comment by EnergyREX** [comment]
Yep, I tried it. This is how I config that. Instead of a cat, I tried a ender_dragon (that was the idea that I was trying at first). I used the following command: /summon_eldritch minecraft:ender_dragon ~ ~ ~ {DragonPhase:1}. Summons the ender dragon, but without any skill of the eldritch mobs mod. 

{

    "values": [

      "minecraft:ender_dragon"

    ] 

}
